### DISCUSSION TITLE
About the boundary handling with MLS in DFSPH

### DISCUSSION THREAD

**Discussion by patricktian1994** [InteractiveComputerGraphics/SPlisHSPlasH#299] [Q&A]
Hi,
Recently I worked on the boundary handling with your great framework, as I see in your paper, the velocity after the divergence-free solver is update with the stiffness parameter, but in this framework it is update with pressure and density like the Band do in the paper.But when it comes to compute the pressure acceleration, no matter with which kind of boundary handling method, it still only considered about the "p_rho2_i" . Is there some advice for this?

**Comment by janbender** [comment]
In our paper [A Survey on SPH Methods in Computer Graphics](https://animation.rwth-aachen.de/publication/0577/) we showed that both formulations are equivalent. A derviation can be found here: [Consistent SPH Rigid-Fluid Coupling](https://animation.rwth-aachen.de/publication/0584/). 

In the implementation I used the term p_i/rho_i^2 (p_rho2_i) which reduces the number of required operations a bit. However, you could also store the pressure value and then recompute the term each time it occurs.
